escape-room/src/main/java/com/escmanager/service/TicketService.java

```diff
@@ -58,6 +58,10 @@ public boolean updateTicket(int id, int user_id, int escape_room_id, BigDecimal
         return true;
     }
 
+    public BigDecimal getMoneyEarned() throws DaoException {
+        return ticketDao.getTotalEarned();
+    }
+
     public List<Ticket> getAllTickets() throws DaoException{
         List<Ticket> ticketList = ticketDao.getAll();
         for (Ticket ticket : ticketList){
```

Comment:
El print tenemos que moverlo en el controller. Yo he creado un metodo en escaperoomMeny y en roomMenu para Escaperoom y room:

public static void printEscapeRooms(List<EscapeRoom> escapeRoomList) {
        for (EscapeRoom escapeRooms : escapeRoomList){
            System.out.println(escapeRooms);
        }
    }